src/main/java/frc/robot/Constants.java

```diff
@@ -27,6 +27,8 @@
     public static final int SHOOTER_MOTOR_1 = 10;
     public static final int SHOOTER_MOTOR_2 = 12;
     public static final int SHOOTER_SOLENOID_PORT = 0;
+    public static final int SHOOTER_ULTRASONIC_TRIG = 3;
```

Comment:
the var name here is `TRIG` but the wpilib api has the first port being for ping and the second for echo. Why not call this `SHOOTER_ULTRASONIC_PING`? What does `TRIG` stand for?

Reply:
I think trig and ping mean the same thing in this case, but the port on the actual device itself is labeled `trig`. I'm guessing it means trigger, but I'm not entirely sure.